src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -51,6 +53,12 @@ protected static ConfigDef baseConfigDef() {
           .define(CONNECTION_URL_CONFIG, Type.LIST, Importance.HIGH,
                   "List of Elasticsearch HTTP connection URLs e.g. ``http://eshost1:9200,http://eshost2:9200``.",
                   group, ++order, Width.LONG, "Connection URLs")
+          .define(CONNECTION_USERNAME, Type.STRING, Importance.LOW,
```

Comment:
Not sure why the `cause` from the exception is getting lost in the test output, but the reason for the error is that both username and password are still required parameters based on this code. This line should be:

```
.define(CONNECTION_USERNAME, Type.STRING, null, Importance.LOW,
```

to define an `null` default -- even if the default is `null`, it still needs to be explicitly specified to indicate there is a default value (and therefore the setting is not required to be specified by the user). Same with the password, which needs a default value to avoid requiring the user to enter a value even if they don't want to use this auth method.